README.md

```diff
@@ -167,5 +171,13 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+lamp = Lamp.new(wishes_left:3)
+genie = Genie.new("Genie")
+lamp.update(genie:genie)
+lamp.update(wishes_remaining:1)
+jafar = Genie.new("Jafar")
+newLamp = Lamp.new(wishes_left:3)
+newLamp.update(genie:jafar)
+newLamp.update(genie:nil)
```

Comment:
:+1: Looks good.  
Be aware, you have updated the lamp before you saved it.  I think that will cause an error.